### PR TITLE
Added another content pointer, added omdb movies api.

### PR DESCRIPTION
**Pull request by amriteyapandey**

### CODE REVIEW

**Comment by amriteyapandey** [comment]
I just noticed this is not open source, just free software. I'll update the PR.
Thanks for pointing out.

File: README.md [hunk 1]
```diff
@@ -214,6 +215,10 @@ APIs
 - [FreeDB](http://www.freedb.org/) - Look up CD information using the freed database.
 - [QuantOneMusic](http://quantonemusic.com/) - Answers the need for digital music data.
 
+### Movies
+
+- [OMDB](https://www.omdbapi.com/) - The OMDb API is a free web service to obtain movie information. ![Open Source](https://raw.githubusercontent.com/Abhishaker17/Awesome-APIs/master/opensource.png?token=AH5Y2imVR5ukgG9533IEqXnP-RJWxVJQks5W4PuNwA%3D%3D "Open Source")
```
Comment (svenluijten) [inline README.md]: This link won't work anymore when you delete your repository. Better [host it on GitHub](https://vinkla.com/2015/upload-images-github/) if you want it to always remain available.

Comment (abhishekbanthia) [inline README.md]: Thanks for pointing out @svenluijten. This is on my to-do.

**Comment by amriteyapandey** [comment]
Hey Abhishek, I have updated the doc. Please have a look.

**Pull request merged**